### PR TITLE
Standard error in map tooltip

### PR DESCRIPTION
**Pull request by ptbrowne**
As for column chart, the map tooltip now show the standard error along
with its unit if the measure dimension has an associated SE dimension.

- Tested with COVID dataset for "-" edgecase
- Tested with Waldfläche dataset for SE in tooltip

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 - Show standard error inside map tooltip
+- Prevent geo coordinates display error when all shapes on a map are deselected
+- Show "-" inside a map tooltip when no value is present
```
Comment (bprusinowski) [inline CHANGELOG.md:12]: Thanks for that!

**Review by bprusinowski — APPROVED** [review]
LGTM 👍

**Comment by Rdataflow** [comment]
checked on https://visualize-ad-feat-map-t-pkbqvb.herokuapp.com/
looks good for the standarderror
unfortunately missing the unit `m3`

**Comment by Rdataflow** [comment]
LGTM :+1:

**Comment by Rdataflow** [comment]
big thx!

**Pull request merged**